runtime/ops/process.rs

```diff
@@ -209,7 +210,7 @@ fn create_command(
   state: &mut OpState,
   args: SpawnArgs,
   api_name: &str,
-) -> Result<std::process::Command, AnyError> {
+) -> Result<(std::process::Command, Option<i32>), AnyError> {
```

Comment:
Ideally this would return `Option<ResourceId>` but we are dealing with file descriptors all the way until `setupChannel` which makes it easier to share code between parent and child fork.